packages/app/src/components/Visualization/__tests__/Visualization.spec.js

```diff
@@ -247,11 +247,10 @@ describe('Visualization', () => {
         vis.instance().recreateChart = recreateChartFn;
 
         it('triggers a reflow when rightSidebarOpen prop changes', () => {
-            vis.setProps({
-                ...props,
-                current: undefined,
-                rightSidebarOpen: true,
-            });
+            // simulate prevProps ?!
```

Comment:
would be great to get a test on the new logic

Reply:
The tests have been updated to account for the new logic, in particular the ones for tDoLoadvisualization.